composer.json

```diff
@@ -17,11 +17,11 @@
     ],
     "extra": {
         "branch-alias": {
-            "dev-master": "2.x-dev"
+            "dev-master": "3.x-dev"
```

Comment:
I've just created a 2.x branch on the repo, thanks for prepping the 3.x-dev as alias already.